LambdaTimeline/View Controllers/CreateAudioCommentViewController.swift

```diff
@@ -0,0 +1,251 @@
+//
+//  CreateAudioCommentViewController.swift
+//  LambdaTimeline
+//
+//  Created by Karen Rodriguez on 5/5/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class CreateAudioCommentViewController: UIViewController {
```

Comment:
This file looks great! I'm tempted to say that you could refactor recorder/player UI into a custom `UIControl` subclass, and the actual recording/playback logic into a custom `AudioRecorderPlayer` class... but that's all definitely secondary. 😅